### DISCUSSION TITLE
onImageDelete event handler

### DISCUSSION THREAD

**Discussion by Sadashii** [outline/rich-markdown-editor#593] [Ideas]
Hello, I use the image upload handler to upload images directly to an aws bucket and currently there is no way or event that I know of which I could trigger when they are deleted from the editor.

**Comment by tommoor** [comment]
There will never be a reliable way to do this in the editor, Instead you should compare the new content against old when you save it

**Reply by Sadashii** [reply]
Is it not possible to add a trigger event on that delete icon?

**Reply by tommoor** [reply]
You could but there are many other ways to remove an image from a document. Eg cut, delete with backspace. Also what happens if the user immediately undoes the change? 

Server side compare is the way to go.
